src/components/NewVaultOfferSummary.tsx

```diff
@@ -110,30 +113,44 @@ const NewVaultOfferSummary = () => {
   const hasErrors =
     collateralizationRatioError || toLockError || toReceiveError;
 
+  const vaultLimitReached = (userVaults?.size ?? 0) >= 150;
```

Comment:
`(userVaults?.size ?? 0)` is used twice. please factor it out to something like `userVaultCount`.

please make the magic numbers 150 and 100 into config constants.